File: VehicleTracker/VehicleTracker/Misc/ShowAlert.m
//
//  ShowAlert.m
//  bdipo
//
//  Created by Nascenia on 3/25/15.
//  Copyright (c) 2015 Nascenia. All rights reserved.
//

#import "ShowAlert.h"

@implementation ShowAlert
+ (void)alertWithMessage:(NSString *)title message:(NSString *)msg{
    UIAlertView *alert = [[UIAlertView alloc] initWithTitle:title message:msg delegate:nil cancelButtonTitle:@"OK" otherButtonTitles:nil];
    [alert show];
}
@end

File: MCP - Swift/MCP/ShowAlert.swift
//
//  ShowAlert.swift
//  bhoganti
//
//  Created by Nascenia on 1/29/15.
//  Copyright (c) 2015 Nascenia. All rights reserved.
//

import UIKit

class ShowAlert {
    class func alertWithMessage(title:String, message:String, buttons:String){
        var alert : UIAlertView = UIAlertView()
        alert.title = title
        alert.message = message
        alert.addButtonWithTitle(buttons)
        alert.show()
    }
}